app/test/unit/git/stash-test.ts

```diff
@@ -46,13 +47,46 @@ describe('git/stash', () => {
       expect(stashEntries[0].branchName).toBe('master')
     })
   })
+
+  describe('createStashEntry', () => {
+    let repository: Repository
+    let readme: string
+
+    beforeEach(async () => {
+      repository = await setupEmptyRepository()
+      readme = path.join(repository.path, 'README.md')
+      await FSE.writeFile(readme, '')
+      await GitProcess.exec(['add', 'README.md'], repository.path)
+      await GitProcess.exec(['commit', '-m', 'initial commit'], repository.path)
+    })
+
+    it('creates a stash entry', async () => {
+      await await FSE.appendFile(readme, 'just testing stuff')
+      const tipSha = await getTipSha(repository)
+
+      await createDesktopStashEntry(repository, 'master', tipSha)
+
+      const result = await GitProcess.exec(['stash', 'list'], repository.path)
+      const entries = result.stdout.trim().split('\n')
+      expect(entries).toHaveLength(1)
```

Comment:
Is there any other assertions you can do here about the results? `exitCode`? Contents of this line?

Reply:
New assertion added - https://github.com/desktop/desktop/pull/7085/commits/df10c173c8f2e000e539dc41fb0b0a52970c26be